modules/odc-deploying-java-applications.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-deploying-java-applications.adoc
+
+[id="odc-deploying-java-applications_{context}"]
+= Upload JAR files for easy deployment of Java applications
+
+You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options: 
+
+* Navigate to *+Add* view of the *Developer* perspective, click the *Upload JAR file* tile through the *From Local Machine* option. Browse and select your JAR file or drag and drop a JAR file to deploy your application.
+
+* Navigate to the *Topology* view and use the *Upload JAR file* option or drag and drop a JAR file to deploy your application.
+
+* Navigate to the Context Menu in the *Topology* view and use the *Upload JAR file* option to upload your JAR file to deploy your application.
+
+Use the following instructions to upload a JAR file in the *Topology* view to deploy a Java application.
+
+.Procedure
+
+. In the *Topology* view, right-click anywhere in the *Topology* view to see the *Add to Project* menu.
+
+. Hover over the *Add to Project* menu to see the menu options, and then select the *Upload JAR file* option to see the *Upload JAR file* form . Alternatively, you can drag and drop the JAR file in the *Topology* view.
+
+. In the *JAR file* field, browse for the required JAR file on your local machine and upload it. Alternatively, you can drag and drop the JAR file on the field. A toast alert is displayed at the top right if an incompatible file type is dragged and dropped on the *Topology* view. On the other hand, a field error is displayed if an incompatible file type is dropped on the field in the upload form.
+
+. You can further specify optional Java commands to customize your deployed application. The Runtime Icon and Builder Image is selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
```

Comment:
capitalization inconsistency. First, you capitalize `Builder Image`. Then, you don't. Is it a formal name? If so, keep it capitalized. If it's not a formal name (so `builder` simply modifies and describe `image`), then don't capitalize it. Same with `Runtime Icon`.

Reply:
Oops! Thanks :rocket: